### PR TITLE
Fix docstring of `src/sage/rings/lazy_series.py`

### PR DESCRIPTION
**Pull request by DavidAyotte**
### 📚 Description

While browsing the documentation, I noticed some minor typesetting issues in `src/sage/rings/lazy_series.py`. This PR aims to fix this.

### 📝 Checklist

- [X] I have made sure that the title is self-explanatory and the description concisely explains the PR.
- [ ] I have linked an issue or discussion.
- [ ] I have created tests covering the changes.
- [X] I have updated the documentation accordingly.

### CODE REVIEW

File: src/sage/rings/lazy_series.py [hunk 1]
```diff
@@ -584,7 +584,7 @@ def map_coefficients(self, f):
 
     def truncate(self, d):
         r"""
-        Return this series with its terms of degree >= ``d`` truncated.
+        Return this series with its terms of degree `\geq` ``d`` truncated.
```
Comment (mantepse) [inline src/sage/rings/lazy_series.py]: I think it would be clearer to write: `Return the series obtained by removing all terms of degree at least ``d``.`

**Review by mantepse — COMMENTED** [review]
excellent, thank you!

File: src/sage/rings/lazy_series.py (hunk 1 above)
Comment (DavidAyotte) [reply src/sage/rings/lazy_series.py]: I agree! I'll change it.

**Review by mantepse — APPROVED** [review]

**Review by dcoudert — APPROVED** [review]
Much better now. Thanks.

**Comment by DavidAyotte** [comment]
Fixes #34610

**Review by mantepse — APPROVED** [review]
thank you!

**Pull request merged**